### PR TITLE
docs: persist theme when using cmdk

### PR DESCRIPTION
**Pull request by sebald**
# Description

Our cmdk did not keep the `?theme=` search param which caused some dumb issues with vercel comments :D 

This makes sure it is always there!

### CODE REVIEW

**Comment by sarahgm** [comment]
Is it intended that its not persist when using cmdk and also a subsection?

**Review by sebald — COMMENTED** [review]
> Is it intended that its not persist when using cmdk and navigating to a subsection?  

@sarahgm nope! rixed

**Comment by sarahgm** [comment]
> > Is it intended that its not persist when using cmdk and navigating to a subsection?
> 
> @sarahgm nope! rixed

you broke the url -> /introduction/design-tokens?theme=corewhat-makes-marigold-differenttypography it doesn't jump to the section

**Comment by sebald** [comment]
@sarahgm third time should be the charm! :D

File: docs/app/_components/CommandItems.tsx [hunk 1]
```diff
@@ -203,11 +203,16 @@ export const PagesItem = ({
   setPages,
 }: PagesItemProps) => {
   const router = useRouter();
-  const goto = (slug: string) => {
-    router.push(`/${slug}`);
+  const params = useSearchParams();
+
+  const goto = ({ slug, hash }: { slug: string; hash?: string }) => {
```
Comment (aromko) [inline docs/app/_components/CommandItems.tsx:208]: `hash` irritates me. I thought first it's a hashed value. I would call it subpage or anchor. Would be more clear to me.

**Review by sarahgm — DISMISSED** [review]

File: docs/app/_components/CommandItems.tsx (hunk 1 above)
Comment (sebald) [reply docs/app/_components/CommandItems.tsx:208]: sadly that is its name :D see https://developer.mozilla.org/en-US/docs/Web/API/URL/hash

**Review by aromko — APPROVED** [review]

**Pull request merged**